index.js

```diff
@@ -71,6 +99,12 @@
 
 // ***** Question 6 *****
 
+function printNameAndPhones(array) {
+  console.log(array[i].name);
+  console.log('Cell: ' + array[i].phones.cell);
+  console.log('Office: ' + array[i].phones.office);
```

Comment:
The idea for this one is to iterate through each user, so the easiest way to do that would be to use a `for..of` or `forEach` loop.

Here's how it would look with `for..of`: 

```js
  for (let user of users) {
    console.log(user.name);
    console.log(`Cell: ${user.phones.cell}`);
    console.log(`Office: ${user.phones.office}`);
  }
```

Here's how it would look with `forEach`: 

```js
  users.forEach(function(user) {
    console.log(user.name);
    console.log(`Cell: ${user.phones.cell}`);
    console.log(`Office: ${user.phones.office}`);
  })
```